package.json

```diff
@@ -90,7 +90,7 @@
       "body-max-line-length": [
         2,
         "always",
-        80
+        90
```

Comment:
What was the reason for this change?

Reply:
This was because one of the collaborators name was longer than 80 characters and could not be properly included in the commit without increasing the limit for each line

Reply:
It doesn't have to do with the user's name, it's only validating the character count of the commit message. It just needed a shorter commit message.

Reply:
Reverted this change